### ISSUE TITLE
Interface Contrast Preference

### ISSUE THREAD

**Issue by jameskoster** [WordPress/gutenberg#54026]
<img width="1490" alt="contrast" src="https://github.com/WordPress/gutenberg/assets/846565/acfe8cde-a18f-4907-a5a4-5a8fde0e2100">

One part of the color scale work (https://github.com/WordPress/gutenberg/issues/53612) worth extracting for separate discussion is an “Interface contrast” user preference.

The _current_ color scale is highly prescriptive about which values components should use based around the objective of meeting WCAG **AA** guidelines. This is a solid default/foundation that should be reflected in the new scale, but it also dismisses large segments of the ability spectrum (for instance those who require a AAA compatible UI). 

A preference for contrast can help individual users create a subjectively comfortable appearance. 

What would be the best way to implement this, if it is agreed that the idea has merit? Should there be a dedicated `ui-element-border` variable the value of which is conditional based in the users contrast preference? Should it be a property of the `Theme` component? Something else? Let's discuss.

cc @WordPress/gutenberg-design 

---

**Note:** The mockup above is a rough sketch, **not a proposal**.

**Comment by jasmussen** [comment]
Rather than a slider, I think we should curate the three options with presets. 

Can the default center option be made identical to what the componentry looks like today?

**Comment by karmatosed** [comment]
I think moving us to feedback would be good for to let it simmer a bit, so let's do that. Having a clear boundary on what can be created will help people then create. It has sat a while and hopefully by doing that we can get discussion going a bit more.
